src/reference/config/writing-platform-configs.rst

```diff
@@ -0,0 +1,207 @@
+
+.. _AdminGuide.PlatformConfigs:
+
+Writing Platform configurations
+===============================
+
+.. versionadded:: 8.0.0
+
+.. seealso::
+
+   - `The original platforms proposal on Cylc Admin <https://github.com/cylc/cylc-admin/blob/master/docs/proposal-platforms.md>`_.
+   - :ref:`Platforms Cylc 7 to 8 user upgrade guide <MajorChangesPlatforms>`.
+   - :cylc:conf:`flow.cylc[runtime][<namespace>]platform`
+   - :cylc:conf:`global.cylc[platforms]`
+
+What are platforms?
+-------------------
+
+Platforms define settings, most importantly a set of hosts and a
+``job runner`` (formerly a ``batch system``) where Cylc can submit a
+task job.
+
+Why were platforms introduced?
+------------------------------
+
+- Allow a compute cluster with multiple login nodes to be treated as a single
+  unit.
+- Allow Cylc to elegantly handle failure of to communicate with login nodes.
+- Configure multiple platforms with the same hosts; for example you can use
+   separate platforms to submit jobs to a batch system and to background on 
+   ``localhost``.
+
+
+Example platforms
+-----------------
+
+Lots of desktop computers
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Platform names are regular expressions.**
+- **you can specify where to install job files.**
+
+.. admonition:: Scenario
+
+   Everyone in your organization has a computer called ``desktopNNN``,
+   all with a file system shared with the scheduler host. Many users
+   will want a platform to run small jobs on their computer:
+
+Cylc treats platform names as regular expressions, so in this case:
+
+.. code-block:: cylc
+   :caption: part of a ``global.cylc`` config file
+
+   [platforms]
+       [[desktop\d\d\d]]
+           install target = localhost
```

Comment:
Would be good to kick things off with `[platforms][localhost]` as the `desktop<pattern>` approach is only necessary if users want to run jobs on targetted desktops.